html/login/views/login.php

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>LOGIN</title>
+        <link rel="stylesheet" type="text/css" href="../public/style.css">
+    </head>
+    <body>
+        <form action="../controllers/authentication.php" method="post">
+            <h2>LOGIN</h2>
+            <?php if (isset($_GET['error'])) { ?>
+                <p class="error"><?php echo $_GET['error']; ?></p>
```

Comment:
```
<p class="error"><?= $_GET['error']; ?></p>
```

nit, for short syntax and much when printing in a html or a template.

always use the shorthand for echoing.

Reply:
Yes sir, i already resolved and pushed it. Thank you.